zulipterminal/ui_tools/buttons.py

```diff
@@ -19,25 +19,31 @@
 from zulipterminal.urwid_types import urwid_Size
 
 
+urwidMarkupTuple = Tuple[Optional[str], str]
```

Comment:
It felt important to include `Tuple` in the name because there are many types of markups and this could form a base for any other markups we create in the future.

Reply:
If we're keeping it in `urwid_types` then an underscore would be good. BTW I think the idea behind the prefix was while there are many types we define for Zulip or internally, these are definitely external and the other names were fairly short and could be confused. We could use `urwid_types.*` in the code but that's more verbose.

Let's keep the original name you had (plus an underscore) - until we define others, we won't have points of comparison, and they risk getting very verbose?